src/main.cpp

```diff
@@ -443,6 +461,10 @@ const unsigned long intervalDisplay = 500;
     #endif
 #endif
 
+#include "powerHandler.h"
+#include "steamHandler.h"
+#include "hardware/brewScale.h"
```

Comment:
We could consider renaming brewScale to scaleHandler

Reply:
therer was a short idea of adding a acaia bluetooth scale, but way in the future :)
better naming schema would be nice

Reply:
The "brew" prefix was supposed to differentiate it from stuff related to scaling as opposed to weighing things. But maybe it's clear from context :)